lua/hbac/command/init.lua

```diff
@@ -0,0 +1,59 @@
+local subcommands = require("hbac.command.subcommands")
+
+local notify_opts = { title = "Hbac" }
+
+local M = {
+	subcommands = {},
+}
+
+M.subcommands.close_unpinned = function()
```

Comment:
If it's possible I would move subcommands to the subcommands.lua file.

Reply:
I wanted to make sure I understood your request correctly before I pushed changes here but I tackled it on another branch

3e6cde0714996e62dd9f7306033231473ade559c

The reason for abstracting/encapsulating the subcommands functions in command/init.lua was to print notifications for `:Hbac` commands but not when Telescope actions were executed. This was admittedly messy. In that branch, I changed the following (and we can take these changes à la carte):

- Moved format_filepath and get_devicon to utils (irrelevant to this issue but I was doing some cleanup for some personal features)
- Add a [notify option](https://github.com/axkirillov/hbac.nvim/blob/3e6cde0714996e62dd9f7306033231473ade559c/lua/hbac/setup.lua#L11) to the user opts so that notifications for `:Hbac` commands could be suppressed (defaults to true, i.e. notifications on)
- Add a hbac_notify function to standardize vim.notify across different modules. If opts.notify is set to false, vim.notify won't be called
- During each Telescope action, opts.notify is set to false before each action, then restored to its original value after with a `set_notify` function in utils

The main benefits are that the command module doesn't use a separate abstracted function just to print a notification, subcommands should be easier to maintain, and changing how notifications are handled is done in one place. 

Another approach I tried instead of toggling the notify opt off and on during each telescope action was to pass a parameter to the hbac_notify function to show/suppress notifications, but I ran into some issues with the attach_mappings Telescope module regarding how bufnr is passed to the toggle_pin function. It would have involved some bigger restructuring I was hesitant to make. Also, since adding the notify opt seemed like a nice feature, I thought it would be better to use that check in one place rather than passing an extra parameter in multiple places.

These were all fairly easy quick changes, so if this isn't what you had in mind and you'd rather save these kinds of changes for a different issue/PR, we can come back around to it. Thanks!

Reply:
Nice! How about we merge this and open a pr for that other branch?

Reply:
Sounds good! Thanks again